buildpipeline/Core-Setup-Linux.json

```diff
@@ -88,8 +88,8 @@
         "definitionType": "task"
       },
       "inputs": {
-        "scriptPath": "build.sh",
-        "args": "$(BuildArguments)",
+        "scriptPath": "scripts/dockerrun-as-current-user.sh",
+        "args": "-t --rm --sig-proxy=true --name $(DockerContainerName) -v $(Build.SourcesDirectory):/opt/code -w /opt/code -e CONNECTION_STRING -e PUBLISH_TO_AZURE_BLOB -e REPO_ID -e REPO_USER -e REPO_SERVER $(PB_EnvVars)$(DockerImageName) /bin/bash -c \"HOME=/opt/code; git clean -X -d -f; ./build.sh $(BuildArguments)\"",
```

Comment:
I would prefer refactoring this so `-e CONNECTION_STRING -e PUBLISH_TO_AZURE_BLOB -e REPO_ID -e REPO_USER -e REPO_SERVER` doesn't duplicate [dockerrun.sh#L138-L154](https://github.com/dotnet/core-setup/blob/53d6b3c1488ffd67888b86a5feda12e3a8a29c84/scripts/dockerrun.sh#L138-L154). There's no need to be specific when specifying these like what you have set up now with `PB_EnvVars` for extras.

To refactor, I think we can remove the existing `docker build` functionality of `dockerrun.sh` and have it take an image name, since we won't need the building feature after this change. `dockerrun-as-current-user.sh` could call `dockerrun.sh`, maybe.